index.js

```diff
@@ -57,6 +57,8 @@ async function fastifyStatic (fastify, opts) {
         : opts.prefix + '/'
   }
 
+  if (opts.constraints === undefined) opts.constraints = {}
+
```

Comment:
I don't think this check is necessary - nor creating an empty object

```suggestion
```

Reply:
That's true, I completely overlooked that undefined is still a valid value for the constraints property. I've removed the check in the most recent commit.